packages/components/src/color-palette/stories/index.tsx

```diff
@@ -33,9 +33,10 @@ export default meta;
 
 const Template: ComponentStory< typeof ColorPalette > = ( {
 	onChange,
+	value,
 	...args
 } ) => {
-	const [ color, setColor ] = useState< string | undefined >();
+	const [ color, setColor ] = useState< string | undefined >( value );
```

Comment:
(Marking this as TODO so we don't forget to address it)

Reply:
@andrewhayward would you happen to know what needed to be done around this line?